### ISSUE TITLE
reconfigure hook fired out of turn

### ISSUE THREAD

**Issue by bodymindarts** [habitat-sh/habitat#1741]
After adding:
```
$ cat hooks/reconfigure
#!/bin/sh
#

echo RECONFIGURE
```
Running the svc will show
```
postgresql.default(SR): Reconfiguration hook failed: hab-sup(ER)[src/error.rs:364:8]: No such file or directory (os error 2)
```
 This only happens on start up before the init script has run the first time.

**Comment by reset** [comment]
Thanks for finding this one, it should be fixed by #1755

**Comment by mwrock** [comment]
closed via #1755
